ui/src/containers/SituationList.vue

```diff
@@ -204,47 +253,76 @@ const onGotoPage = (nextPage: number) => {
 			No results found
 		</div>
 	</div>
-	<FeatherSnackbar v-model="showError" center error>
-		{{ appStore.errorMessage || 'Error has occurred :(' }}
-	</FeatherSnackbar>
 </template>
 
 <style lang="scss" scoped>
 @import '@/styles/variables.scss';
 .list-main {
 	min-height: 800px;
 }
+
+.reset-btn {
+	font-size: 12px;
+	padding: 0px 12px;
+	margin-right: 20px;
+}
+
+.icon {
+	font-size: 17px;
+	margin-right: 5px;
+}
+
 h2 {
 	margin-top: 0;
 	margin-bottom: 15px !important;
 }
-.situation-filters {
+
+.autocomplete {
+	margin-left: auto;
+}
+.filters {
 	padding: 15px;
-	border: 1px solid #dfdfdf;
 	margin-bottom: 20px;
 	background-color: #ffffff;
 	margin-top: 10px;
+	display: flex;
+	flex-direction: row;
```

Comment:
Small thing, but likely you won't need to specify this line here or below as any `display: flex` will be a `row` by default